src/shared/directives/conditional-show/conditional-show-form.directive.ts

```diff
@@ -95,16 +88,11 @@ export class ConditionalShowFormDirective implements OnInit, AfterViewInit, OnDe
           cf.hidden = false;
         }
         if (condResult === cf.hidden) {
-          if (cf.hidden) {
-            this.greyBarService.addToggledToShow(cf.id);
-          } else {
-            this.greyBarService.removeToggledToShow(cf.id)
-          }
```

Comment:
hope there is reason to remove this logic

Reply:
@rayliangatos - this logic was deleted by @paul-graham in commit ebea39b in PR #764

Reply:
@rayliangatos, @Daniel-Lam:

Yes, the `GreyBarService` is no longer used for displaying the grey bars - an entirely different approach is in place. This was done as part of https://tools.hmcts.net/jira/browse/EUI-3267 and some tidying up happened subsequently in https://tools.hmcts.net/jira/browse/EUI-3854.